### PR TITLE
Manipulation: Restrict the tbody search to child nodes

### PR DESCRIPTION
**Pull request by gibson042**
Fixes gh-3439

```
   raw     gz Sizes                                                            
267269  79153 dist/jquery.js                                                   
 86863  30052 dist/jquery.min.js                                               

   raw     gz Compared to master @ bf3a43eff8682b59cec785be6003753fa4b93706    
   +59    +19 dist/jquery.js                                                   
    -1     +9 dist/jquery.min.js
```

### CODE REVIEW

File: src/manipulation.js [hunk 1]
```diff
@@ -47,11 +47,12 @@ var
 	rscriptTypeMasked = /^true\/(.*)/,
 	rcleanScript = /^\s*<!(?:\[CDATA\[|--)|(?:\]\]|--)>\s*$/g;
 
+// Append table rows to a tbody when one is present
 function manipulationTarget( elem, content ) {
 	if ( jQuery.nodeName( elem, "table" ) &&
 		jQuery.nodeName( content.nodeType !== 11 ? content : content.firstChild, "tr" ) ) {
 
-		return elem.getElementsByTagName( "tbody" )[ 0 ] || elem;
+		return jQuery.filter( "tbody", elem.childNodes )[ 0 ] || elem;
```
Comment (markelog) [inline src/manipulation.js]: Clever, maybe a comment though? You know I like comments :)

**Comment by markelog** [comment]
@timmywil just FYI (in case you didn't know that): you can now approve pull requests, it seems a bit better then "reaction" :)

File: src/manipulation.js (hunk 1 above)
Comment (gibson042) [inline src/manipulation.js]: Hmm, what are you looking for that's not covered by the new function comment? Should I update it to something like "New rows for a table belong in its child tbody when one is present", or move it above this line, or something else?

Comment (dmethvin) [inline src/manipulation.js]: Actually, the code itself looks pretty clear and definitely has the benefit of being more correct. Funny that nobody thought about nested tables for this case.

Comment (gibson042) [inline src/manipulation.js]: It only comes up when the target doesn't have a tbody but a descendant does, or when such a descendant is in the thead, both of which I imagine are rare. And the logic itself goes [_waaaaaay_ back](https://github.com/jquery/jquery/commit/7c04a64016c1f295dfdf4e731f693f8400a987bc#diff-169760a97de5c86a886842060321d2c8R163).

**Pull request merged**